apps/fabric-website/src/pages/Controls/PersonaPage/docs/ios/PersonaOverview.md

```diff
@@ -1,5 +1,11 @@
-Consistent representation of people within our experience helps users connect with others effortlessly. Fabric has a couple of controls available to illustrate people accurately and consistently.
+Representing users in a consistent manner across our products help with effortless communication. Using Fabric Persona controls ensure an accurate and consistent experience across Microsoft products.
```

Comment:
@bigbadcapers - I know you know this, but the language in some of these pieces reads internal-facing, e.g. "our products"... "ensure an accurate and consistent experience across **Microsoft products**", etc. That's not necessarily wrong, but doesn't feel like the most appropriate tone for a public-facing library.

An alternate version of this might look like: 
> Representing users in a consistent manner across different apps help with effortless communication.

That said, let's not block on that for this PR since a number of the current controls follow this pattern, and we should do a sweep for tone/voice consistency after Build.